### ISSUE TITLE
.Net 6

### ISSUE THREAD

**Issue by Webreaper** [hypnopotamus/BlazorReact#15]
If you're interested, I've created a .Net 6 version of this project/idea: 

https://github.com/Webreaper/BlazorReact

**Comment by hypnopotamus** [comment]
hey, that's awesome! I'll try to take some time this weekend to check it out.

Did you notice any significant differences with .net 6 and the most recent Blazor packages?  I created this repo to go along with this article https://chrdyks.medium.com/blazor-webassembly-and-typescript-react-in-one-application-from-a-net-developer-f2e7529aea76 and then haven't made any significant updates so that I don't need to worry about updating the article's instructions or explanations... but, of course, as time moves on framework and package versions keep moving and my repo stays the same as you're already plenty aware.

**Comment by Webreaper** [comment]
Yeah, I found the repo via your article - as it was something I was trying to do for work. I tried upgrading your repo but it didn't work (unsure why) so instead I retro-fitted the scripts and csproj bits into a clean new Blazor template project, and it all worked....
